src/App.css

```diff
@@ -0,0 +1,82 @@
+.nav {
+  height: 7vh;
```

Comment:
- [ ] [Optional] You can improve the design by making sure the header has a fixed height. Please refer to the image below.

![image](https://user-images.githubusercontent.com/92215878/221000383-8275e366-de83-41c7-939a-b84c4e1937bb.png)